src/contrib/cluster/Akka.Cluster.Sharding/ShardingMessages.cs

```diff
@@ -22,6 +23,18 @@ public interface IShardRegionCommand { }
     /// </summary>
     public interface IShardRegionQuery { }
 
+    /// <summary>
+    /// Used as a special termination message from <see cref="ClusterSharding"/>
```

Comment:
Might want to specify that this is only for `Shard` actors

Reply:
changed it slightly